granary/test/testdata/comment.mf2.json

```diff
@@ -1,5 +1,5 @@
 {
-  "type": ["h-entry", "p-comment"],
+  "type": ["h-entry", "h-as-comment"],
```

Comment:
oh i see, nm. the `comment` property stores comments *on* this object? so `h-as-comment` is all we want?